editor/plugins/script_text_editor.cpp

```diff
@@ -1454,9 +1455,9 @@ void ScriptTextEditor::drop_data_fw(const Point2 &p_point, const Variant &p_data
 			}
 
 			if (preload) {
-				text_to_drop += "preload(\"" + String(files[i]).c_escape() + "\")";
+				text_to_drop += "preload(" + quote_style + String(files[i]).c_escape() + quote_style + ")";
```

Comment:
Note that you could use `String::quote()` too: https://github.com/godotengine/godot/blob/a8fb450b3ce568fa8e3c9c013821f9046b9a0396/core/string/ustring.h#L308